src/types/service.ts

```diff
@@ -0,0 +1,5 @@
+export type UBID = number[];
+export type Addr = number[];
+
+// TOOD fix this
+export const DISCOVERY = [0];
```

Comment:
add new line, I'd add this to the `discovery` file too.

Reply:
> add new line

What?

>I'd add this to the discovery file too.

It's imported?

Reply:
File doesn't have a new line at the end of it, all others do.

I wouldn't import, i'd put it there so its all in one